machine-learning/solutions/seismic_salt_detection/trainer/model.py

```diff
@@ -73,51 +73,40 @@ def create_regressor(config, parameters):
     Returns:
         A configured and ready to use `tf.estimator.DNNRegressor`
     """
-
     # Mean and Standard Deviation Constants for normalization.
-    with file_io.FileIO(parameters.mean_path, mode='r') as f:
-        mean = pickle.load(f)
-    with file_io.FileIO(parameters.std_path, mode='r') as f:
-        std = pickle.load(f)
+    mean = np.float32(parameters.depth_mean)
+    std = np.float32(parameters.depth_std)
 
     # Columns to be used as features.
-    hour = tf.feature_column.categorical_column_with_identity(
-        'hour', num_buckets=24)
-    hour = tf.feature_column.embedding_column(
-        hour, dimension=parameters.hour_embedding)
-
-    day = tf.feature_column.categorical_column_with_identity(
-        'day', num_buckets=7)
-    day = tf.feature_column.embedding_column(
-        day, dimension=parameters.day_embedding)
-
-    weather = [tf.feature_column.numeric_column(
-        'weather' + str(i),
-        normalizer_fn=(lambda x, i = i: (x - mean[i]) / std[i])
-    ) for i in range(constants.WEATHER_SIZE)]
-
-    distribution = [tf.feature_column.numeric_column(
-        'distribution' + str(i)
-    ) for i in range(constants.DISTRIBUTION_SIZE)]
-
-    feature_cols = [hour, day] + weather + distribution
+    
+    depth = tf.feature_column.numeric_column(
+        'depth',
+        normalizer_fn=(lambda x: (x - mean) / std))
+    
+    image = hub.image_embedding_column('image', parameters.module)
+    
+    feature_cols = [depth,image]
 
-    # Evaluation metric.
-    def mean_absolute_error(labels, predictions):
-        """Creates mean absolute error metric.
+    def estimator_metrics(labels, predictions):
```

Comment:
Why is this function nested inside create_regressor?  I don't see any variables being closed.

Reply:
:-( They are not following the style guide.

https://engdoc.corp.google.com/eng/doc/devguide/py/style/index.md#264-decision

"Avoid nested functions or classes except when closing over a local value. Do not nest a function just to hide it from users of a module. Instead, prefix its name with an _ at the module level so that it can still be accessed by tests.

Reply:
will fix